src/main/java/edu/ucar/unidata/rosetta/controller/TemplateController.java

```diff
@@ -695,21 +929,22 @@ private String getIpAddress(HttpServletRequest request) {
         }
         return ipAddress;
     }
-
+*/
     /**
      * Creates a unique id for the file name from the clients IP address and the
      * date.
      *
      * @param request The HttpServletRequest.
      * @return The unique file name id.
      */
+    /*
     private String createUniqueId(HttpServletRequest request) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)